<launch>
    <node name="nmpc_pc_learning" pkg="nmpc_pc_learning" type="nmpc_pc_learning" output="screen" />

    <param name="verbose" type="bool" value="true" /> <!-- Parameter to display NMPC messages.-->
    <param name="yaw_control" type="bool" value="true" /> <!-- Parameter to let NMPC control yaw angle.-->
    <param name="online_ref_yaw" type="bool" value="true" /> <!-- Parameter to utilize online reference for yaw angle. -->
    <param name="use_dist_estimates" type="bool" value="false" /> <!-- Parameter to let NMPC use disturbace estimates.-->
    <param name="mocap_topic_part" type="string" value="mocap" /> <!-- Parameter to specify part of the topic name for localization data. -->
    <param name="dist_Fx_predInit_topic" type="string" value="gp_disturb_reg/predInit/x" /> <!-- Parameter to specify the topic name for dist_Fx_predInit. -->
    <param name="dist_Fy_predInit_topic" type="string" value="gp_disturb_reg/predInit/y" /> <!-- Parameter to specify the topic name for dist_Fy_predInit. -->
    <param name="dist_Fz_predInit_topic" type="string" value="gp_disturb_reg/predInit/z" /> <!-- Parameter to specify the topic name for dist_Fz_predInit. -->
    <param name="dist_Fx_data_topic" type="string" value="gp_disturb_reg/mu/x" /> <!-- Parameter to specify the topic name for dist_Fx_data. -->
    <param name="dist_Fy_data_topic" type="string" value="gp_disturb_reg/mu/y" /> <!-- Parameter to specify the topic name for dist_Fy_data. -->
    <param name="dist_Fz_data_topic" type="string" value="gp_disturb_reg/mu/z" /> <!-- Parameter to specify the topic name for dist_Fz_data. -->



    <!-- <include file="$(find nmpc_pc_learning)/launch/airframes/quad_F330.xml" /> -->
    <!-- <include file="$(find nmpc_pc_learning)/launch/airframes/quad_diatone.xml" /> -->
    <!-- <include file="$(find nmpc_pc_learning)/launch/airframes/quad_martianVI.xml" /> -->
    <!-- <include file="$(find nmpc_pc_learning)/launch/airframes/matrice_100.xml" /> -->
    <include file="$(find nmpc_pc_learning)/launch/airframes/matrice_100_gazebo.xml" />
    <!-- <include file="$(find nmpc_pc_learning)/launch/airframes/tricopter_talon.xml" /> -->
    <!-- <include file="$(find nmpc_pc_learning)/launch/airframes/tricopter_white.xml" /> -->
    <!-- <include file="$(find nmpc_pc_learning)/launch/airframes/tricopter_microscale.xml" /> -->

</launch>
